pkg/devfile/adapters/kubernetes/component/adapter.go

```diff
@@ -160,6 +160,30 @@ func (a Adapter) Push(parameters common.PushParameters) (err error) {
 	}
 	s.End(true)
 
+	log.Info("\nUpdating services")
```

Comment:
Please add a spinner for this section otherwise it might feel unresponsive

Reply:
I've added a spinner when waiting for the pod to be in a non-ready state. The update of the services is done in the devfile only, do you think it needs a spinner?

Reply:
The call `PushServiceFromKubernetesInlineComponents` seems to make calls to the cluster like `CreateDynamicResource` and `DeleteDynamicResource`.

Reply:
Yes, you're right, I changed to embed the call to PushServiceFromKubernetesInlineComponents in the spinner. Thanks

Reply:
It seems adding the spinner causes overlapping on the output
```
Updating services
 ◓  Updating services ✓  Deleted service "AtlasCluster/atlascluster" from the cluster
 ✓  Updating services [4s]
```
Let's skip the spinner for now.